test/e2e/apps/job.go

```diff
@@ -112,21 +114,45 @@ var _ = SIGDescribe("Job", func() {
 	// Pods sometimes fail, but eventually succeed, after pod restarts
 	ginkgo.It("should run a job to completion when tasks sometimes fail and are not locally restarted", func() {
 		ginkgo.By("Creating a job")
-		// 50% chance of container success, local restarts.
+		// Success and failure alternate, no local restarts.
 		// Can't use the failOnce approach because that relies
 		// on an emptyDir, which is not preserved across new pods.
```

Comment:
rather than an emptyDir or a http callback, can we force the jobs to be scheduled to a single node and use a hostPath volume instead of an emptyDir? cc @kubernetes/sig-storage-misc

Reply:
Good idea, I have updated in this way and verified it can work. PTAL.